pkgs/development/libraries/pvsneslib/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gnumake
+, cmake
+, gcc
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pvsneslib";
+  version = "4.2.0";
+
+  src = fetchFromGitHub {
+    owner = "alekmaul";
+    repo = "pvsneslib";
+    rev = version;
+    hash = "sha256-Cl4+WvjKbq5IPqf7ivVYwBYwDDWWHGNeq4nWXPxsUHw=";
+    fetchSubmodules = true;
+  };
+
+  patches = [ ./fixed_makefiles.patch ];
+
+  nativeBuildInputs = [ gnumake cmake gcc ];
+
+  configurePhase = ''
+    echo "No configurePhase"
```

Comment:
Changed.

Disabled configurePhase because CMake try to configure something (root has no CMakefile) but cmake is required for compiler/wla-dx

Removed unused nativeBuildInputs

Reply:
Then you should try `dontUseCmakeConfigure = true;`